delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/JsonLdError.scala

```diff
@@ -0,0 +1,46 @@
+package ch.epfl.bluebrain.nexus.delta.rdf.jsonld
+
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.RemoteContextResolutionError
+
+@SuppressWarnings(Array("IncorrectlyNamedExceptions"))
+sealed abstract class JsonLdError(reason: String) extends Exception {
```

Comment:
We could have here directly the underlying exception

Reply:
This is the way we have been defining errors on KG, CLI and everywhere else